code/Model/SiteTree.php

```diff
@@ -760,14 +760,9 @@ public function isSection()
      */
     public function isOrphaned()
     {
-        // Always false for root pages
-        if (empty($this->ParentID)) {
```

Comment:
Retain this check

`return !$parent` below means we've flipped the logic here where a root page (with no parent) will mistakenly return `true` for `isOrphaned()` when it should return `false`